src/plugins/kibana_usage_collection/server/collectors/event_loop_delays/rollups/integration_tests/daily_rollups.test.ts

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import type { Logger, ISavedObjectsRepository } from '../../../../../../../core/server';
+import {
+  createTestServers,
+  TestElasticsearchUtils,
+  TestKibanaUtils,
+  createRootWithCorePlugins,
+} from '../../../../../../../core/test_helpers/kbn_server';
+import { rollDailyData } from '../daily';
+import { mocked } from '../../event_loop_delays.mocks';
+
+import {
+  SAVED_OBJECTS_DAILY_TYPE,
+  serializeSavedObjectId,
+  EventLoopDelaysDaily,
+} from '../../saved_objects';
+import moment from 'moment';
+
+const { startES } = createTestServers({
```

Comment:
optional: `createTestServers` returns kibanaServer object, there is no need to import `createRootWithCorePlugins`